python/spec/dependabot/python/update_checker/requirements_updater_spec.rb

```diff
@@ -370,7 +388,7 @@
           context "when the requirement version is too high" do
             let(:setup_cfg_req_string) { ">=2.0.0" }
 
-            its([:requirement]) { is_expected.to eq(:unfixable) }
+            its([:requirement]) { is_expected.to eq(">=1.5.0") }
```

Comment:
I like the previous outcome better here. If the latest resolvable version is 1.5.0, and user has specified `>=2.0.0` in their manifest, I think either:

* User setup/manifest is wrong.
* Dependabot run into a bug when figuring out the latest resolvable version.

Either way, it seems best to leave this kind of ill defined case alone?

Also, the new solution is actually widening the requirement, not increasing it, which seems completely against the spirit of this PR?